src/sentry/api/serializers/models/group_stream.py

```diff
@@ -153,11 +153,13 @@ def query_tsdb(self, groups: Sequence[Group], query_params, **kwargs):
         except Environment.DoesNotExist:
             stats = {g.id: tsdb.make_series(0, **query_params) for g in groups}
         else:
+            org_id = groups[0].project.organization_id if groups else None
```

Comment:
Assumes all groups in the list belong to a single org, let me know if this assumption is incorrect.

Reply:
that assumption seems correct. is `groups` guaranteed to have at least one element though?

Reply:
ah right, `if groups` will evaluate to `False` if it is empty.